Documentation/ceph-disaster-recovery.md

```diff
@@ -336,3 +336,17 @@ Assuming `dataHostPathData` is `/var/lib/rook`, and the `CephCluster` trying to
 1. Bring the Rook Ceph operator back online by running `kubectl -n rook-ceph edit deploy/rook-ceph-operator` and set `replicas` to `1`.
 1. Watch the operator logs with `kubectl -n rook-ceph logs -f rook-ceph-operator-xxxxxxx`, and wait until the orchestration has settled.
 1. **STATE**: Now the new cluster should be up and running with authentication enabled. `ceph -s` output should not change much comparing to previous steps.
+
+## Adopting into a new Kubernetes cluster with PVCs
+
+It is possible to migrate/restore an rook/ceph cluster from an exising Kubernetes cluster to a new one without restoriting to SSH access or ceph tooling. This allows doing the migration using standard kubernetes resources only. This guide assumes you have a CephCluster that uses PVCs to persist mon and osd data.
+1. Stop rook in the cluster cluster by scaling the operator deployment `rook-ceph-operator` down to zero and deleting the other deployments: `rook-ceph-mgr-a`, `rook-ceph-mon-a`, ..., `rook-ceph-osd-0`, ...
+1. Restore the rook PVCs to the new cluster. A common way to do this is to generate regular snapshots of the volumes and re-create PVCs from these volumes in the new cluster.
+1. Copy the keyring and fsid secretsfrom the old cluster: `rook-ceph-mgr-a-keyring`, `rook-ceph-mon`, `rook-ceph-mons-keyring`, `rook-ceph-osd-0-keyring`, ...
+1. Delete mon services and copy them from the old cluster: `rook-ceph-mon-a`, `rook-ceph-mon-b`, ... Note the simply re-applying won't work because the goal here is to restore the `clusterIP` in each service and this field is immutable in `Service` resources.
+1. Copy the endpoints configmap from the old cluster: `rook-ceph-mon-endpoints`
+1 Scale the operator back up and wait until the reconciliation is over.
+
+
```

Comment:
How about showing kubectl or other commands that will help users get this working? Right now the document is high level and may be ambiguous for how to backup and restore the resources.

Reply:
I see your point that the exact commands will depend on the tools. This is an advanced scenario, so it seems acceptable to leave the details to the reader.

Reply:
Actually I did add several kubectl commands as examples..